pkg/storage/aws/cloudfront/options.go

```diff
@@ -7,7 +7,7 @@ import (
 // Option represents the Cloudfront storage options
 type Option func(s *Storage)
 
-// WithCloudfrontHost sets the cloudfront URL
+// WithCloudfrontHost sets the cloudfront host, can ends up with trailing slash or not
```

Comment:
Should be `... can end up ...`

Reply:
lol you are right, thanks for the correction